login-workflow/package.json

```diff
@@ -25,17 +25,17 @@
     "prettier": "@pxblue/prettier-config",
     "dependencies": {
         "@pxblue/react-auth-shared": "^3.3.0-beta.0",
-        "dompurify": "^2.0.12"
+        "dompurify": "^2.2.9"
     },
     "peerDependencies": {
-        "@material-ui/core": "^4.7.2",
-        "@pxblue/colors": "^3.0.0",
-        "@pxblue/react-components": "^5.1.0",
-        "date-fns": "^2.14.0",
+        "@material-ui/core": "^4.11.4",
+        "@pxblue/colors": "^3.0.1",
+        "@pxblue/react-components": "^5.2.0",
+        "date-fns": "^2.22.1",
         "i18next": "^19.3.4 || ^20.0.0",
         "react": "^16.13.1 || ^17.0.0",
         "react-dom": "^16.8.6 || ^17.0.0",
-        "react-i18next": "^11.3.4",
+        "react-i18next": "^11.10.0",
         "react-router-dom": "^5.2.0"
```

Comment:
So we don't actually want to update these peerDependencies unless the older versions do not work (i.e. if somebody has 2.14.2 of date-fns we don't want to force them to update to 2.22.1 unless we are using something specifically from 2.22.1). We should be able to simply update the dev dependencies (and update our lock file to be using the latest versions that satisfy the minimum requirements).